content/en/serverless/installation/go.md

```diff
@@ -163,13 +277,25 @@ go get github.com/DataDog/datadog-lambda-go
     }
     ```
 
+[1]: https://app.datadoghq.com/organization-settings/api-keys
+{{% /tab %}}
+{{< /tabs >}}
+### Install the Datadog Lambda library
```

Comment:
Why is this section outside of the tabs, but the instructions for using it inside the tabs?